assets/prototype/domain/shared/predicates/dateUtilities/sortDates.ts

```diff
@@ -0,0 +1,18 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type DatesAreEqualType = {
+	dates: Date[];
+	order: 'asc' | 'desc';
+};
+
+const sortDates = ({ dates, order }: DatesAreEqualType) => {
+	if (order === 'asc') {
+		return dates.sort(compareAsc);
+	}
+
+	if (order === 'desc') {
+		return dates.sort(compareDesc);
+	}
```

Comment:
Here is the same approach as in `date-fns`' docs:
![image](https://user-images.githubusercontent.com/1477453/70986846-672ffb80-20c7-11ea-9841-0415f7095f97.png)
*** 
https://date-fns.org/v2.8.1/docs/compareAsc